frontend/src/components/donation_page/index.jsx

```diff
@@ -1,39 +1,39 @@
 import React, { Component } from "react";
+import { Container, Row, Col } from "react-bootstrap";
```

Comment:
`Col` is not being used -> so lets remove that from here

Reply:
Done